helm/ffc-pay-enrichment/templates/config-map.yaml

```diff
@@ -9,15 +9,19 @@ data:
   PROCESSING_TOPIC_ADDRESS: {{ .Values.container.processingTopicAddress }}-{{ .Values.environmentCode }}
   RESPONSE_TOPIC_ADDRESS: {{ .Values.container.responseTopicAddress }}-{{ .Values.environmentCode }}
   EVENT_TOPIC_ADDRESS: {{ .Values.container.eventTopicAddress }}-{{ .Values.environmentCode }}
+  EVENTS_TOPIC_ADDRESS: {{ .Values.container.eventsTopicAddress }}-{{ .Values.environmentCode }}
```

Comment:
Correct.

The service bus topics in each environment have a suffix of either `-dev`, `-test`, `-pre` or `prd`.  Rather than have to keep adding every topic name to app config, we can use some wizardry in the Helm chart so we just need a single `environmentCode` value instead.